apps/nrm/views/details-last-contact.html

```diff
@@ -0,0 +1,16 @@
+{{<partials-page}}
+{{$page-content}}
+<p class="govuk-body">{{#t}}pages.details-last-contact.paragraph-1{{/t}}</p>
+<p class="govuk-body">{{#t}}pages.details-last-contact.paragraph-2{{/t}}</p>
+<p class="govuk-body">{{#t}}pages.details-last-contact.paragraph-3{{/t}}</p>
+
+<h2>Details of the last contact (optional)</h2>
```

Comment:
This can also be extracted into a markdown file if the content becomes complex (more than a few paragraphs)